added autoconf to apt install requirements (RPi)

sfinktah opened the pull request:
I found I needed autoconf/automake.  It may be possible to avoid this by adjusting the timestamps of the related files, but simply installing it takes < 30 seconds, and avoid this and any future issues.

I also added `sudo -s` after the `reboot` (see previous commit), and changed the initial `sudo su` to `sudo -s` for consistency. 

```
config.status: executing libtool commands
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /home/pi/nexmon/buildtools/mpfr-3.1.4/missing aclocal-1.15 -I m4
/home/pi/nexmon/buildtools/mpfr-3.1.4/missing: line 81: aclocal-1.15: command not found
WARNING: 'aclocal-1.15' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
```

conor-f commented:
This isn't fully complete. See #344 for details. Basically just also needs the packages `texinfo`, `libtool-bin`, and `automake` then run `autoreconf -f -i` in the `mpfr` directory. If I get time I'll just patch this onto this PR.

baselsayeh commented:
@sfinktah 
the source setup_env.sh do not need root access

sfinktah commented:
I fully appreciate that this was far from a ideal solution, but I thought it worth documenting some instructions that at least worked.

@baselsayeh I don't believe I added any root access that wasn't already in there, just changed `su` for `sudo`
